packages/astro-font/AstroFont.astro

```diff
@@ -7,10 +7,8 @@ const { config } = Astro.props as Props
 const resolvedConfig = await generateFonts(config)
 
 const preloads = resolvedConfig.map(createPreloads)
-const baseCSS = Promise.all(resolvedConfig.map(createBaseCSS))
-const fallbackCSS = Promise.all(resolvedConfig.map(createFontCSS))
+const styles = Promise.all([...resolvedConfig.map(createBaseCSS), ...resolvedConfig.map(createFontCSS)])
 ---
 
 {preloads.flat().map((content) => <link as="font" crossorigin rel="preload" href={content} type={getPreloadType(content)} />)}
-{baseCSS.then((res) => res.flat().map((content) => <style set:html={content} is:inline />)).catch(console.log)}
-{fallbackCSS.then((res) => res.flat().map((content) => <style set:html={content} is:inline />)).catch(console.log)}
+{styles.then((res) => <style set:html={res.flat().reduce((total, content) => `${total}\n${content}`, '')} is:inline />).catch(console.log)}
```

Comment:
`.reduce((total, content) => total ? `${total} ${content}` : content, '')`

how about this to make sure no extra spacing?